Properties/AssemblyInfo.cs

```diff
@@ -19,9 +19,9 @@
 // NOTE: the version info is read by a regex in a script so don't add complicated things to this file
 // NOTE: When changing the version number you need to also change "export_presets.cfg"
 
-[assembly: AssemblyVersion("0.5.6.1")]
+[assembly: AssemblyVersion("0.5.7.0")]
 
-[assembly: AssemblyInformationalVersion("")]
+[assembly: AssemblyInformationalVersion("alpha")]
```

Comment:
doesn't this need to be "-alpha"? Or am I misremembering?

Reply:
Apparently it is.